backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ComponentDatabaseHandler.java

```diff
@@ -419,6 +449,15 @@ private Vendor getVendor(String vendorId) {
     ///////////////////////////////
 
     public RequestStatus updateComponent(Component component, User user) throws SW360Exception {
+        removeLeadingTrailingWhitespace(component);
```

Comment:
I am not sure about this particular part, the trim whitespace option will be maybe difficult to extend if particular fields are referenced like this example here